index.html

```diff
@@ -491,7 +491,7 @@ <h2>Ballerina in Action</h2>
         Album album = check self.db->queryRow(`SELECT * FROM Albums WHERE id=${id}`);
         if currency != self.baseCurrency {
             string query = string `from=${self.baseCurrency}&to=${currency}`;
-            record {decimal rate;} exchange = check self.forex->get(string `/curerncyConversion?${query}`);
+            record {decimal rate;} exchange = check self.forex->get(string`/curerncyConversion?${query}`);
```

Comment:
shall we remove this change.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/3788/commits/4be38dab70259bb1ddf68d3ef91bad314af87e7e